shared/test/test_sound_library_api.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'test_helper'
+require 'sound_library_api'
+
+class SoundLibraryTest < Minitest::Test
+  include Rack::Test::Methods
+  include SetupTest
+
+  def build_rack_mock_session
+    @session = Rack::MockSession.new(SoundLibraryApi, 'studio.code.org')
+  end
+
+  def test_get_sound
+    stubs(:cache_for).returns(nil)
+    get '/api/v1/sound-library/test_sound.mp3'
+    assert last_response.ok?
+  end
+
+  def test_get_deleted_sound
+    stubs(:cache_for).returns(nil)
+    get '/api/v1/sound-library/test_sound.mp3'
+    assert last_response.ok?
```

Comment:
A more robust version of this test (which I'm not necessarily recommending for this PR) would interact with the S3 API directly and handle its own test files.  In pseudocode, something like this:

```
# (In a setup step) Assert that no version of test file exists on S3
# Upload test file with contents "A" to S3
# Assert that sound library API returns test file with contents "A"
# Overwrite test file with contents "B" on S3
# Assert that sound library API returns test file with contents "B"
# Delete test file from S3
# Assert that sound library API returns test file with contents "B"
# (In a cleanup step) Remove all versions of test file from S3
# (In a cleanup step) Assert that no version of test file exists on S3
```

This would let the test correctly re-generate its own fixtures, should we ever need to do so, and might communicate the test's intent in code instead of in comments.